packages/eva/mapping.json

```diff
@@ -1124,6 +1124,356 @@
+            "type": "number"
+          },
+          "placeholderLineHeight": {
+            "type": "number"
+          },
+          "placeholderFontWeight": {
+            "type": "string"
+          },
+          "placeholderColor": {
+            "type": "string"
+          },
+          "textMarginHorizontal": {
+            "type": "number"
+          },
+          "textFontSize": {
+            "type": "number"
+          },
+          "textLineHeight": {
+            "type": "number"
+          },
+          "textFontWeight": {
+            "type": "string"
+          },
+          "textColor": {
+            "type": "string"
+          },
+          "iconWidth": {
+            "type": "number"
+          },
+          "iconHeight": {
+            "type": "number"
+          },
+          "iconTintColor": {
+            "type": "string"
+          },
+          "iconMarginHorizontal": {
+            "type": "number"
+          },
+          "menuMaxHeight": {
+            "type": "number"
+          },
+          "menuBorderRadius": {
+            "type": "number"
+          },
+          "menuBorderColor": {
+            "type": "string"
+          },
+          "menuBorderWidth": {
+            "type": "number"
+          },
+          "labelColor": {
+            "type": "string"
+          },
+          "labelFontSize": {
+            "type": "number"
+          },
+          "labelLineHeight": {
+            "type": "number"
+          },
+          "labelFontWeight": {
+            "type": "string"
+          },
+          "labelMarginBottom": {
+            "type": "number"
+          },
+          "outlinePadding": {
+            "type": "number"
+          },
+          "outlineBorderRadius": {
+            "type": "number"
+          },
+          "outlineBackgroundColor": {
+            "type": "string"
+          }
+        },
+        "appearances": {
+          "default": {
+            "default": true
+          }
+        },
+        "variantGroups": {
+          "status": {
+            "primary": {
+              "default": true
+            },
+            "success": {
+              "default": false
+            },
+            "info": {
+              "default": false
+            },
+            "warning": {
+              "default": false
+            },
+            "danger": {
+              "default": false
+            }
+          }
+        },
+        "states": {
+          "disabled": {
+            "default": false,
+            "priority": 0,
+            "scope": "all"
+          },
+          "active": {
+            "default": false,
+            "priority": 1,
+            "scope": "all"
+          }
+        }
+      },
+      "appearances": {
+        "default": {
+          "mapping": {
+            "menuMaxHeight": 220,
+            "menuBorderRadius": 4,
+            "menuBorderColor": "border-basic-color-4",
+            "menuBorderWidth": 1,
+            "controlBorderRadius": 4,
```

Comment:
It's better to use these parameters without prefixes in case it is applied to a box

Reply:
Refactored